src/index.d.ts

```diff
@@ -327,3 +357,6 @@ interface OverflowableTextProps {
 }
 
 export const OverflowableText: FunctionComponent<OverflowableTextProps>;
+export { ROW_DRAGGING_SELECTION_COLUMN_DEF } from './components/react-hook-form/ag-grid-table-rhf/custom-ag-grid-table';
+export { FILTER_EQUIPMENTS } from './components/filter/utils/criteria-based-utils';
+export { CONTINGENCY_LIST_EQUIPMENTS } from './components/filter/utils/criteria-based-utils';
```

Comment:
There is 2 sections please respect it for clarity.

```
/**
 * Section to export generated type declarations
 */
```

```
/**
 * Section to export manual type declarations of .js and .jsx files
 */
```

If it's a TS file, it's necessary in the first section.